### PR TITLE
Implement ui for selecting route to event

### PR DESCRIPTION
**Pull request by Sosol02**

### CODE REVIEW

File: app/src/androidTest/java/com/github/onedirection/map/MapFragmentTest.java [hunk 1]
```diff
@@ -524,6 +527,61 @@ public void onRequestMade() {
...
+        onView(withId(R.id.fragment_map_event_nav_route_button))
+                .perform(click());
+        onView(withId(R.id.fragment_map_ui))
+                .perform(new WaitAction(1000));
+
+
+
+        //NavigationManager navigationManager = getFragmentField("navigationManager", NavigationManager.class);
+        //com.mapquest.navigation.NavigationManager navigationManager1 = getAttributeField("navigationManager", navigationManager, com.mapquest.navigation.NavigationManager.class);
+        //assertThat(navigationManager1.getNavigationState(), equalTo(com.mapquest.navigation.NavigationManager.NavigationState.ACTIVE));
```
Comment (TheImpressiveDonut) [inline app/src/androidTest/java/com/github/onedirection/map/MapFragmentTest.java:572]: Should you leave that?

**Review by TheImpressiveDonut — CHANGES_REQUESTED** [review]
Looks good.

File: app/src/androidTest/java/com/github/onedirection/map/MapFragmentTest.java (hunk 1 above)
Comment (Sosol02) [reply app/src/androidTest/java/com/github/onedirection/map/MapFragmentTest.java:572]: It was for the other test that i was about to write, i will remove it ofc.

**Review by TheImpressiveDonut — APPROVED** [review]

**Review by justagoodguy-omega — APPROVED** [review]
All good, and I am very grateful that this was not my task

**Pull request merged**